README.md

```
@@ -1,39 +1,97 @@
+  - Google Drive → Credential Manager + Google Identity Services + Drive REST v3
+- **Local file access**: Storage Access Framework (`DocumentFile` /
+  `DocumentsContract`) — no `MANAGE_EXTERNAL_STORAGE`
 
-```bash
-yarn create next-app --example https://github.com/vercel-labs/ai-sdk-preview-internal-knowledge-base ai-sdk-preview-internal-knowledge-base-example
+## Module / package layout
+
+Currently a single-module project (`:app`) with a clean package layout so it can
+be split into Gradle modules later without code churn:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)